utils/middlewares.js

```diff
@@ -1,8 +1,24 @@
 const morgan = require("morgan");
 const express = require("express");
+const cors = require("cors");
+const helmet = require("helmet");
 
-module.exports = function(app) {
+const CORS_ORIGIN_WHITELIST = ['http://localhost:3000'];
```

Comment:
We need to add hosted front-end `URL` to the white list

Reply:
We should also check for the environment for `dev` or `prod` in order to know which options to use

Reply:
Yes, the front-end url will be added.

For the check of the environment, I do not think doing that will add any extra ability to this method.